database/migrations/2020_05_20_175045_add_foreign_key_group_id_to_users_table.php

```diff
@@ -14,7 +14,7 @@ class AddForeignKeyGroupIdToUsersTable extends Migration
     public function up()
     {
         Schema::table('users', function (Blueprint $table) {
-            $table->foreign('group_id')->references('id')->on('groups')->onDelete('cascade');
+            $table->foreign('group_id')->references('id')->on('groups')->onDelete('set null');
```

Comment:
WTF is this??

Reply:
According to the initial requirement, the testees linked to the group should be removed with it.